activesupport/test/logger_test.rb

```diff
@@ -120,14 +135,57 @@ def test_buffer_multibyte
     byte_string.force_encoding("ASCII-8BIT")
     assert byte_string.include?(BYTE_STRING)
   end
-  
+
   def test_silencing_everything_but_errors
     @logger.silence do
       @logger.debug "NOT THERE"
       @logger.error "THIS IS HERE"
     end
-    
+
     assert !@output.string.include?("NOT THERE")
     assert @output.string.include?("THIS IS HERE")
   end
+
+  def test_logger_thread_safety
+    @logger.level = Logger::INFO
+
+    assert @logger.level == Logger::INFO,
```

Comment:
Ah, I see. Because the `level` returns an integer and not a symbol. I'd rather we added a custom and private assertion, like `assert_level`.